pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -182,17 +180,19 @@ func loadMonConfig(clientset kubernetes.Interface, namespace string) (map[string
 	return monEndpointMap, maxMonID, monMapping, nil
 }
 
-func createCSISecret(clientset kubernetes.Interface, namespace string, clusterInfo *cephconfig.ClusterInfo, ownerRef *metav1.OwnerReference) error {
-	var err error
+func createCSISecret(context *clusterd.Context, namespace string, clusterInfo *cephconfig.ClusterInfo, ownerRef *metav1.OwnerReference) error {
+	// check if the secret is present
+	// if not create the key
+	csiSecretKey, err := createCSIKeyring(context, namespace, ownerRef)
 
 	// Store the admin secret for the csi driver
 	csiSecrets := map[string][]byte{
 		// adminID is expected for the cephfs driver
 		"adminID":  []byte("admin"),
```

Comment:
You're updating this one as well? One of these is for cephfs and the other is for the rbd driver. Looks like one of the comments is incorrect since they're not both for cephfs.

Reply:
I just updated what @dillaman suggested. Also, I thought the comments were weird too but I could find which one was correct or not, so I just let it like it was.